cfgov/unprocessed/js/modules/post-filter.js

```diff
@@ -107,24 +107,24 @@ PostFilter.prototype = {
     var fromDateValue = this.$gte.val();
     var toDateValue = this.$lte.val();
     var dateRange = dateRangeFormatter.format( fromDateValue, toDateValue );
-    var error_messages = [];
+    var errorMessages = [];
 
     if ( dateRange && dateRange.isValid ) {
       this.$gte.val( dateRange.startDate );
       this.$lte.val( dateRange.endDate );
     } else {
-      error_messages.push( DATE_ERRORS.invalid );
+      errorMessages.push( DATE_ERRORS.INVALID );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/8a828a9fea878b0aaa9975c3a914611071f25e88